wallet/src/main/res/values/strings.xml

```diff
@@ -141,6 +145,13 @@
     <string name="consent_prompt_title_verifier">%1$s is requesting information from \"%2$s\"</string>
     <string name="consent_prompt_icon_description">Verifier Icon</string>
 
+    <!-- Biometric Prompt -->
+    <string name="biometric_prompt_default_title">User Authentication</string>
+    <string name="biometric_prompt_negative_btn_lskf">Use PIN</string>
+    <string name="biometric_prompt_negative_btn_no_lskf">Cancel</string>
+    <string name="biometric_prompt_subtitle_biometric">Log in using your biometric credential</string>
+    <string name="biometric_prompt_subtitle_lskf">Log in using your PIN</string>
```

Comment:
Why are we showing the user "Log in using your [...]" here? What are they logging into? What could we be showing the user instead?

Reply:
I mean, "authenticate" is more accurate than "log in" but maybe a little scarier-sounding

Reply:
I just went ahead and removed subtitles from the prompts since you mentioned space is a concern and the UI indicates what we're asking the user for

Reply:
OK, I think we want a subtitle but we can revisit this later.